### PR TITLE
Fixed an OAuth 2.0 Authorization issue in ServiceNow V2 

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [46055](https://github.com/demisto/etc/issues/46055)

## Description
- Fixed an issue where the call that is in charge of getting an attachment file that is attached to a 'ServiceNow' ticket, isn't compatible with a client that is configured with OAuth 2.0 Authorization.
- Added a unit test testing that issue
- Added the 'servicenow-get-ticket' command to the 'playbook-ServiceNow_OAuth_Test'.
- Remove 'servicenow-oauth-login' command from the 'playbook-ServiceNow_OAuth_Test'.

**The edited TPB:** 
ServiceNow_OAuth_Test_Sun_Jan_30_2022

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
In my first commit I added a unit test that is failing due to the bug described in the PR's body. 
You can see the build failure [here](URL).

File: Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py [hunk 1]
```diff
@@ -32,6 +32,12 @@ def __init__(self, credentials: dict, use_oauth: bool = False, client_id: str =
             self.password = credentials.get('password')
             self.auth = (self.username, self.password)
 
+        if '@' in client_id:  # for use in OAuth test-playbook
+            self.client_id, refresh_token = client_id.split('@')
+            integration_context = get_integration_context()
```
Comment (Itay4) [inline Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py:37]: is there a reason for that? will there be anything in the context?

File: Packs/ServiceNow/ReleaseNotes/2_2_23.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ServiceNow v2
+- Fixed the ***servicenow-get-ticket*** command to correctly authorize the request when getting an attachment from a 'ServiceNow' ticket.
```
Comment (Itay4) [inline Packs/ServiceNow/ReleaseNotes/2_2_23.md:4]:
```suggestion
- Fixed an issue where the ***servicenow-get-ticket*** command failed when to get attachments when using *OAuth Login*.
```

**Review by Itay4 — APPROVED** [review]
great work 👍

File: Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py (hunk 1 above)
Comment (ShacharKidor) [reply Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py:37]: Changed that. thanks!

**Pull request merged**